src/ServiceControl/Infrastructure/RavenDB/RavenBootstrapper.cs

```diff
@@ -48,7 +48,7 @@ public void StartRaven(EmbeddableDocumentStore documentStore, Settings settings,
             Directory.CreateDirectory(settings.DbPath);
 
             documentStore.DataDirectory = settings.DbPath;
-            documentStore.UseEmbeddedHttpServer = maintenanceMode || settings.ExposeRavenDB;
+            documentStore.UseEmbeddedHttpServer = true;// maintenanceMode || settings.ExposeRavenDB;
```

Comment:
I would suggest so that we revert this change and use @gbiellem [solution](https://github.com/Particular/ServiceControl/pull/934#issuecomment-286667354)